sdk/communication/azure-communication-administration/README.md

```diff
@@ -184,6 +184,135 @@ def purchase_search():
     )
 ```
 
+##Communication Phone number
+### Get Countries
+
+```python
+def list_all_supported_countries():
+    phone_number_administration_client = PhoneNumberAdministrationClient.from_connection_string(connection_str)
+    
+    supported_countries = phone_number_administration_client.list_all_supported_countries()
+    for supported_country in supported_countries:
```

Comment:
you have this functions removed in other PR. I would suggest updating this only in one of the PRs